google/iam/credentials/v1/iamcredentials.proto

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019 Google LLC.
```

Comment:
I think this is part of the IAM Service Account Credentials API and should not be included in api-common